class/getPosts.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function () {
+  // To initially set all counts of links.
+  $(".like-count").each(function () {
+    $userId = $(this).parent().parent().parent().attr("id").split("-")[0];
+    $postId = $(this).attr("id").substr(11);
+    $(this).load("../Model/updateLike.php?postId=" + $postId + " #totalLikes");
+  });
+
+  // To update status of likes.
+  $.get("../Model/updateLike.php?&userId=" + $userId, function ($r) {
+    $.each($r.split(" "), function ($key, $val) {
+      if ($val != "") {
+        $("#like-" + $val).addClass("liked");
+        $("#like-" + $val).attr("liked", "TRUE");
+      }
+    });
+  });
+
+  // To update like counts in DB.
```

Comment:
use multiline comments for functions.

Reply:
I don't think inline functions need specific multiline comments.